bcgov/biohubbc
# BHBC 2009

# Overview

## Links to Jira tickets

[BHBC-2009](https://quartech.atlassian.net/browse/BHBC-2009?atlOrigin=eyJpIjoiNzQxNmU2MDYwMmZlNDFmMTlmNTdiNjVmMWEwMjMxNTgiLCJwIjoiaiJ9)

## Description of relevant changes

Added a function wrapper for the database class to catch all exceptions thrown.
To test: 
- create a project
- create a survey
- open survey page in two tabs
- make changes and save in one tab
- make changes and save in second tab
- see stale project data error instead of 'Unexpected Error'

## PR Checklist

A list of items that are good to consider when making any changes.

_Note: this list is not exhaustive, and not all items are always applicable._

### Code


### Style


### Documentation


### Tests


### Linting/Formatting

       _See the `lint` commands in package.json_
       _See the `format` commands in package.json_

### SonarCloud
